src/manifolds/Grassmann.jl

```diff
@@ -88,6 +88,32 @@ function allocation_promotion_function(::Grassmann{n,k,ℂ}, f, args::Tuple) whe
     return complex
 end
 
+@doc raw"""
+    change_representer(M::Grassmann, ::EuclideanMetric, p, X)
+
+Change `X` to the corresponding representer of a cotangent vector at `p`.
+Since the [`Grassmann`](@ref) manifold `M`, is isometrically embedded, this is the identity
+"""
+change_representer(::Grassmann, ::EuclideanMetric, ::Any, ::Any)
+
+function change_representer!(M::Grassmann, Y, ::EuclideanMetric, p, X)
+    copyto!(M, Y, p, X)
```

Comment:
I haven't seen this signature of `copyto!` before. Where is it defined?

Reply:
The first one at https://juliamanifolds.github.io/ManifoldsBase.jl/stable/functions.html#Base.copyto!-Tuple{AbstractManifold,%20Any,%20Any,%20Any} (for vectors).
I just got very used to this, because it also takes care about types and structs; for Grassmann this should still work fine for the `Projector` representation for example.
I started using it when I worked with FixedRank matrices where each point/tangent is stored as 3 arrays and copying a tangent vectors requires the point as well (if I remember correctly).
For manifolds that work on pure arrays this defaults to just `copyto!(Y,X)`.

Reply:
IMO code restricted to a particular manifold should just use the two-arg version of `copyto!` because it's easier for the compiler. These manifold specialisations of `copyto!` are only relevant to certain variants of power manifolds.

Reply:
I'm fine with this version for now. In the future we could adopt a consistent rule and make sure it is followed throughout the codebase.

Reply:
Sure it might be a good idea to work a bit on a consistent usage here